install_config/install/advanced_install.adoc

```diff
@@ -1181,6 +1181,108 @@ xref:../../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#i
+ifdef::openshift-enterprise[]
+openshift_service_catalog_image_prefix=openshift3/ose-
+endif::[]
+ifdef::openshift-origin[]
+openshift_service_catalog_image_prefix=openshift/origin-
+openshift_service_catalog_image_version=latest
+endif::[]
+----
+
+When the service catalog is enabled, the web console shows the updated landing
+page but still uses the normal image stream and template behavior. To deploy and
+register the provided service brokers, see
+xref:configuring-ansible-service-broker[Configuring the Ansible Service Broker]
+and xref:configuring-template-service-broker[Configuring the Template Service Broker].
+
+[[configuring-ansible-service-broker]]
+=== Configuring the Ansible Service Broker
+
+[NOTE]
+====
+The Ansible service broker is currently a Technology Preview feature.
+====
+
+If you have xref:enabling-service-catalog[enabled the service catalog], you can
+also enable the
+xref:../../architecture/service_catalog/ansible_service_broker.adoc#arch-ansible-service-broker[Ansible service broker] (ASB).
+
+The ASB requires its own etcd instance separate from the etcd instance used by
+the {product-title} cluster, as well as separate storage using persistent
+volumes (PVs). Additional PVs are also required for the Ansible playbook bundles
+(APBs) that are created using the ASB. The following example uses NFS to provide
+the persistent storage.
+
+To enable the ASB and define its storage requirements, add the following in your
+inventory file's `[OSEv3:vars]` section:
+
+----
+ifdef::openshift-enterprise[]
+ansible_service_broker_image_prefix=openshift3/
+ansible_service_broker_registry_url="registry.access.redhat.com"
+endif::endif::[]
+ifdef::openshift-origin[]
+ansible_service_broker_image_prefix=openshift/
+ansible_service_broker_registry_url="registry.access.redhat.com"
+endif::[]
+
+openshift_hosted_etcd_storage_kind=nfs
+openshift_hosted_etcd_storage_nfs_directory=/opt/osev3-etcd
+openshift_hosted_etcd_storage_volume_name=etcd-vol2
+openshift_hosted_etcd_storage_access_modes=['ReadWriteOnce']
+openshift_hosted_etcd_storage_volume_size=10G
+openshift_hosted_etcd_storage_host=<ip_address>
+openshift_hosted_etcd_storage_labels="{'storage': 'etcd'}"
+----
+
+[[configuring-template-service-broker]]
+=== Configuring the Template Service Broker
+
+[NOTE]
+====
+The template service broker is currently a Technology Preview feature.
+====
+
+If you have xref:enabling-service-catalog[enabled the service catalog], you can
+also enable the
+xref:../../architecture/service_catalog/template_service_broker.adoc#arch-template-service-broke[template service broker] (TSB).
+
+To enable the TSB:
+
+. One or more projects must be defined as the broker's source
+namespace(s) for loading templates and image streams into the service catalog.
+Set the desired projects by modifying the following in your inventory file's
+`[OSEv3:vars]` section:
++
+----
+openshift_template_service_broker_namespaces=['openshift','myproject']
```

Comment:
no, this is for the TSB.  We don't full expose the TSB as part of the installer, but we do make it possible to at least configure it (so that if you do want to expose it, it's ready to go).  That's what this does: sets up the part of the master-config that "turns on" the TSB (but no one can call it due to permissions restrictions).  @jim-minter added the behavior to the installer.